inc/profile.class.php

```diff
@@ -1479,6 +1489,48 @@ function showFormSetup($openform = true, $closeform = true) {
    }
 
 
+   /**
+    * Print the helpdesk form for a profile
+    *
+    * @param $openform     boolean  open the form (true by default)
```

Comment:
You've missed the `@since 9.4.0` tag, and the `@return`. For other params, syntax is `@param type $name A bit of documentation` ;)